dashboard/app/controllers/api/v1/schools_controller.rb

```diff
@@ -7,4 +9,14 @@ def index
     end
     render json: serialized_schools
   end
+
+  # GET /dashboardapi/v1/schoolsearch/:criteria/:limit
+  def search
+    schools = School.where("name LIKE ?", "%#{params[:q].upcase}%").order(:name).limit(params[:limit])
+    #ChatClient.log "#{schools.to_s}"
+    serialized_schools = schools.map do |school|
+      Api::V1::SchoolSerializer.new(school).attributes
+    end
+    render json: serialized_schools
+  end
```

Comment:
It probably makes sense to validate that `params[:limit]` isn't too large, otherwise this method could bring down an FE / our DB.